src/Components/test/testassets/TestServer/Infrastructure/InterruptibleWebSocket.cs

```diff
@@ -0,0 +1,75 @@
+using System;
+using System.IO;
+using System.Net.WebSockets;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Components.TestServer.Infrastructure
```

Comment:
Is this a new namespace? Can we avoid hiding things in rando namespaces if it is

Reply:
It causes me a lot of grief that we fight the IDE, can we stop doing that or set the namespace in the project or something that makes the default right? if not, why fight it?

I don't know what the right solution it is here, btw.

Reply:
I did update it this time, but my concern is still valid.